Makefile.am

```diff
@@ -128,3 +129,14 @@ dist_extratmpfiles_DATA = \
 ldconfdir = $(sysconfdir)/ld.so.conf.d
 dist_ldconf_DATA = \
 	ld.so.conf.d/flatpak-extension-runtime-fdo-openh264.conf
+
+com.endlessm.TestMode.policy: com.endlessm.TestMode.policy.in
+	$(AM_V_GEN) $(SED) -e 's|[@]sbindir@|$(sbindir)|g' < "$<" > "$@"
+
+policydir = $(POLKIT_POLICYDIR)
+dist_policy_DATA = \
```

Comment:
Since the policy file is generated at build time, it shouldn't be distributed. Otherwise it will have the `sbindir` path from one the tarball was made. In other words, just `policy_DATA`.

Reply:
Good catch. Fixed.